code/game/gamemodes/cult/cult_items.dm

```diff
@@ -54,14 +54,17 @@
 	force = 15
 	throwforce = 25
 	embed_chance = 75
+	nemesis_factions = list("ratvar")
 
 /obj/item/weapon/melee/cultblade/dagger/attack(mob/living/target, mob/living/carbon/human/user)
 	..()
 	if(iscarbon(target))
 		var/mob/living/carbon/C = target
 		C.bleed(50)
-		if(is_servant_of_ratvar(C) && C.reagents)
-			C.reagents.add_reagent("heparin", 1)
+
+/obj/item/weapon/melee/cultblade/dagger/nemesis_effects(mob/living/user, mob/living/target)
+	if(iscarbon(target) && target.reagents)
+		target.reagents.add_reagent("heparin", 1) //servants and ratvar-aligned carbons bleed profusely
```

Comment:
Holy fuck. Why would you use heparin to make them bleed? It makes way more sense just to use the bleed proc...